src/indexmap.jl

```diff
@@ -68,28 +68,22 @@ function ITensors.inds(imap::IndexMap)
   @assert keys(index_dimension(imap)) == keys(index_digit(imap))
   return collect(keys(index_dimension(imap)))
 end
-function dimension_inds(imap::IndexMap, dimension::Int)
-  return collect(
-    filter(i -> index_dimension(imap)[i] == dimension, keys(index_dimension(imap)))
-  )
+function dimension_inds(imap::IndexMap, dim::Int)
+  return collect(filter(i -> index_dimension(imap)[i] == dim, keys(index_dimension(imap))))
 end
-function ind(imap::IndexMap, dimension::Int, digit::Int)
+function ind(imap::IndexMap, dim::Int, digit::Int)
   return only(
     filter(
-      i -> index_dimension(imap)[i] == dimension && index_digit(imap)[i] == digit,
+      i -> index_dimension(imap)[i] == dim && index_digit(imap)[i] == digit,
       keys(index_dimension(imap)),
     ),
   )
 end
 
-function dimension_indices(imap::IndexMap, dimap::Int)
-  return filter(ind -> dimension(imap, ind) == dim, inds(imap))
-end
```

Comment:
Removed this function... its not using anywhere, and `dimap` is not used inside the function...